backends/ebpf/run-ebpf-test.py

```diff
@@ -154,42 +184,50 @@ def run_test(options, argv):
     result, expected_error = ebpf.compile_p4(argv)
 
     # Compile and run the generated output
-    if result == SUCCESS and not expected_error:
+    if result == testutils.SUCCESS and not expected_error:
         # If no empty.stf present, don't try to run the model at all
         if not os.path.isfile(testfile):
             msg = "No stf file present!"
-            report_output(output["stdout"],
-                          options.verbose, msg)
-            result = SUCCESS
+            testutils.log.info(msg)
+            result = testutils.SUCCESS
         else:
             result = run_model(ebpf, testfile)
     # Only if we did not expect it to fail
-    if result != SUCCESS:
+    if result != testutils.SUCCESS:
         return result
 
     # Remove the tmp folder
     if options.cleanupTmp:
-        report_output(output["stdout"],
-                      options.verbose, "Removing", tmpdir)
-        shutil.rmtree(tmpdir)
+        testutils.del_dir(options.testdir)
     return result
 
 
-if __name__ == '__main__':
-    """ main """
+if __name__ == "__main__":
+    """main"""
```

Comment:
```suggestion
```

not a very useful documentation... (and probably not even working for code blocks)